src/elm/Page/Community.elm

```diff
@@ -611,7 +614,11 @@ viewAction loggedIn metadata maybeDate action =
                         [ if validationType == "CLAIMABLE" then
                             button
                                 [ class ("h-10 uppercase rounded-lg ml-1" ++ claimColors ++ claimSize)
-                                , onClick (OpenClaimConfirmation action.id)
+                                , if isClosed == False then
```

Comment:
- `isClosed` is already a Boolean, so it's better to use the value itself with no comparison (`if isClosed` or `if not isClosed`).
- Also, `onClick` is different than `class`, and it's better to separate them.
- The `class ""` in `else` branch looks like a trick to cheat the type system :) We have `NoOp` variant in `Msg` type to reflect an empty message, so it's better to use it instead (this pattern is already used in the file in several places).

I would write something like this:

```elm
onClick
    (if isClosed then
        NoOp

     else
        OpenClaimConfirmation action.id
    )
```

Reply:
+1

Reply:
Good catch :)